pyaedt/modeler/cad/Modeler.py

```diff
@@ -1134,6 +1207,388 @@ def _attributes(self):
         return coordinateSystemAttributes
 
 
+class ObjectCoordinateSystem(BaseCoordinateSystem, object):
+    """Manages object coordinate system data and execution.
+
+    Parameters
+    ----------
+    modeler :
+        Inherited parent object.
+    props : dict, optional
+        Dictionary of properties. The default is ``None``.
+    name : optional
+        The default is ``None``.
+    entity_id : int
+        Id of the entity object where the Object Coordinate System is anchored.
```

Comment:
```suggestion
        The default is ``None``.
    entity_id : int
        ID of the entity object where the object coordinate system is anchored.
```
Avoid unnecessary capitalization--reserve for proper nouns or matching tagged code entities. Don't forget to add a description for name.